### PR TITLE
Add Mannan's Hackbar

### PR DESCRIPTION
**Pull request by Mannan2812**

### CODE REVIEW

File: _hackbars/Mannan_Bansal.md [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+---
+
+OpenSource is full of new innovations and learning opportunities. What are you waiting for, start contributing.
+
+---
+
+## Humans are the only animal who will cut tree, make paper and write save trees on them. #Hacktoberfest #SaveEarth #StopDeforestation
+
+
+![image_caption](../assets/images/Mannan_Bansal.jpg)
```
Comment (tasnimzotder) [inline _hackbars/Mannan_Bansal.md]: The path is relative to the website not only to the directory. Also, don't forget to put a caption.

```suggestion
![image_caption]({{site.baseurl}}/assets/images/Mannan_Bansal.jpg)
```

Comment (Mannan2812) [reply _hackbars/Mannan_Bansal.md]: Ok I will update it.

Comment (Mannan2812) [reply _hackbars/Mannan_Bansal.md]: Hey Buddy I have query when I put the path that way it just does not render when we preview the md file. Is it Normal or am I doing something wrong?

**Comment by Mannan2812** [comment]
I have added necessary changes and What about this WIP emoji in front of title ??

**Comment by tasnimzotder** [comment]
WIP - Work in progress

**Review by tasnimzotder — APPROVED** [review]

File: _hackbars/Mannan_Bansal.md (hunk 1 above)
Comment (tasnimzotder) [reply _hackbars/Mannan_Bansal.md]: The `.md` file doesn't render itself. The website uses Jekyll to generate the static page. And the paths are relative to the root website.

**Pull request merged**